client/asset/dcr/dcr.go

```diff
@@ -38,20 +37,65 @@ import (
 )
 
 const (
-	BipID      = 42
+	// BipID is the BIP-0044 asset ID.
+	BipID = 42
+	// defaultFee is the default value for the fallbackfee.
 	defaultFee = 20
+	// splitTxBaggage is the total number of additional bytes associated with
+	// using a split transaction to fund a swap.
+	splitTxBaggage = dexdcr.MsgTxOverhead + 1 + dexdcr.P2PKHInputSize + 2*(1+dexdcr.P2PKHOutputSize)
```

Comment:
@chappjc, you've recently done some work with these size calculations, so please check me on these values. I'm adding the push opcode sizes here too. Is that right?

Reply:
Referring to the `1 +` on the input and output sizes?  At the level of txn serialization, it's not Script data push opcodes, but the varint encoding as done by `wire.WriteVarInt`.  But `P2PKHInputSize` and `P2PKHOutputSize` both already have the varint accounted assuming the inputs' sig script sizes and outputs' pkScript sizes are less than 253 bytes, so I think the tx size should just be `dexdcr.MsgTxOverhead + dexdcr.P2PKHInputSize + 2*dexdcr.P2PKHOutputSize`.